tests/apm/monitoring-commandFailed-001.phpt

```diff
@@ -41,17 +41,23 @@ $subscriber = new MySubscriber;
 
 MongoDB\Driver\Monitoring\addSubscriber( $subscriber );
 
-drop_collection(URI, DATABASE_NAME, COLLECTION_NAME);
+$primary = get_primary_server(URI);
+$command = new \MongoDb\Driver\Command(['geoNear' => COLLECTION_NAME]);
+try {
+    $primary->executeCommand(DATABASE_NAME, $command);
+} catch (Exception $e) {
+    /* Swallow */
+}
 ?>
 --EXPECT--
-started: drop
-failed: drop
+started: geoNear
+failed: geoNear
```

Comment:
`geoNear` is deprecated ([PM-939](https://jira.mongodb.org/browse/PM-939)). I think a more robust solution would just be to issue an `aggregate` with an unsupported pipeline operator (e.g. `[ { "$unsupported": 1 } ]`) or an `update` with a similarly unsupported modifier.